website/docs/cli/commands/console.mdx

```diff
@@ -9,6 +9,14 @@ description: >-
 The `tofu console` command provides an interactive console for
 evaluating [expressions](../../language/expressions/index.mdx).
 
+:::note
```

Comment:
Nitpick, but sometimes this note is before the "Usage" section, and sometimes it is in the "Usage" section right before the flags are mentioned.
Let's be consistent and choose a single place. I think it makes more sense to include this section inside the  "Usage" section, as it's relevant to how you use the command.

Reply:
Yep, makes sense. Moved note to usage block for most of the commands: [3980be3](https://github.com/opentofu/opentofu/pull/1843/commits/3980be38e72560ca2a97aea35f51c0f0ea66b9cf)

Reply:
that's a miss, thank you!

[e27f67c](https://github.com/opentofu/opentofu/pull/1843/commits/e27f67c2a6ed5485312283c102c84e11accd337b)